### ISSUE TITLE
install-snapshot: cannot use repo with owner name attached (liquibase/liquibase-pro)

### ISSUE THREAD

**Issue by StevenMassaro** [liquibase/liquibase-sdk-maven-plugin#2]
If I do:

```
mvnDebug -B liquibase-sdk:install-snapshot "-Dliquibase.sdk.branchSearch=master" "-Dliquibase.sdk.repo=liquibase-pro" "-Dliquibase.sdk.github.token=...
```

This succeeds with the latest unreleased code.

If I do:

```
mvnDebug -B liquibase-sdk:install-snapshot "-Dliquibase.sdk.branchSearch=master" "-Dliquibase.sdk.repo=liquibase/liquibase-pro" "-Dliquibase.sdk.github.token=...
```

I get this error:

```
[INFO] --- liquibase-sdk-maven-plugin:0-SNAPSHOT:install-snapshot (default-cli) @ s3-remote-accessor-extension ---
[INFO] Looking for master from a run in liquibase/liquibase-pro
[INFO] Found matching branch: liquibase-pro:master
[INFO] ----Listening for transport dt_socket at address: 8000
--------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  59.944 s
[INFO] Finished at: 2022-08-03T08:01:23-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.liquibase.ext:liquibase-sdk-maven-plugin:0-SNAPSHOT:install-snapshot (default-cli) on project s3-remote-accessor-extension: Unknown repo: liquibase/liquibase-pro -> [Help 1]
[ERROR]
```

which is thrown from logic that does not handle the owner name here: https://github.com/liquibase/liquibase-sdk-maven-plugin/blob/main/src/main/java/liquibase/sdk/maven/plugins/InstallSnapshotMojo.java#L60-L67.

**Comment by StevenMassaro** [comment]
Perhaps this is actually supposed to work this way, which is fine with me, but then the readme needs to be updated under the https://github.com/liquibase/liquibase-sdk-maven-plugin#install-snapshot header, because it explicitly says that this is permitted.

**Comment by nvoxland** [comment]
This should be fixed in #3
